patches/tModLoader/Terraria/ModLoader/ModItem.cs

```diff
@@ -671,11 +671,21 @@ public abstract class ModItem : ModTexturedType
 		public virtual void OpenBossBag(Player player) {
 		}
 
+		/// <summary>
+		/// Allows you to decide if this item is allowed to stack with another of its type.
+		/// <br>This is only called when attempting to stack with an item of the same type.</br>
+		/// <br>This is usually not called for coins and ammo in the inventory/UI.</br>
+		/// <br>This covers all scenarios, if you just need to change in-world stacking behavior, use <see cref="CanStackInWorld"/></br>
+		/// </summary>
```

Comment:
This is how breaks are supposed to be used.. Can you apply to this to the rest?
```suggestion
		/// Allows you to decide if this item is allowed to stack with another of its type.
		/// <br/>This is only called when attempting to stack with an item of the same type.
		/// <br/>This is usually not called for coins and ammo in the inventory/UI.
		/// <br/>This covers all scenarios, if you just need to change in-world stacking behavior, use <see cref="CanStackInWorld"/>
		/// </summary>
```